eslint.config.js

```diff
@@ -0,0 +1,25 @@
+import { includeIgnoreFile } from '@eslint/compat';
+import { fileURLToPath } from 'node:url';
+import ts from 'typescript-eslint';
+
+import svelte from 'eslint-plugin-svelte';
+
+import { storyblokLintConfig } from '@storyblok/eslint-config';
+
+const gitignorePath = fileURLToPath(new URL('./.gitignore', import.meta.url));
+
+export default storyblokLintConfig(
+  includeIgnoreFile(gitignorePath),
+  {
+    ignores: ['playground', '**/node_modules/**', 'README.md', 'LICENSE'],
```

Comment:
We should not ignore the playground for the linter, the indentation is even different there @edodusi

Reply:
oh ok ok, its gonna be done in a separate PR then?

Reply:
no I will add them here after we discuss the possibility of changing the public API of the lib, ATM the DX is exactly the same but I'm brainstorming some possible updates that I will share privately